Misc/NEWS

```diff
@@ -327,6 +327,9 @@ Extension Modules
 Library
 -------
 
+- bpo-30378: Fix the problem that logging.handlers.SysLogHandler cannot
```

Comment:
Yes, NEWS now is the main source of merge backport conflicts, but we need to log what happened. :-(